frontend/src/App.tsx

```diff
@@ -25,6 +29,12 @@ export const App = () => {
           <Route path=":traineeId" element={<TraineeDetails />} />
         </Route>
         <Route path="/instruktorzy" element={<InstructorsList />} />
+        <Route path="/pojazdy">
+          <Route index element={<VehiclesList />} />
+          <Route path="new" element={<VehicleNew />} />
+          <Route path=":vehicleId/edit" element={<VehicleEdit />} />
```

Comment:
if '/pojazdy' in polish, than '/edytuj' and not edit, no?

Reply:
Great catch! Changed this and `new` → `nowy`